### PR TITLE
Changelog 56 next

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add entry for next release
Version to be discussed 
cc @sbesson

### CODE REVIEW

**Review by mtbc — APPROVED** [review]

File: history.rst [hunk 1]
```diff
@@ -5,6 +5,24 @@
 OMERO version history
 =====================
 
+5.6.2 (July 2020)
+-----------------
+
+This release adds installation documentation for server and web on
+CentOS 8 and Ubuntu 20.04, and it removes installation instructions
+for Ubuntu 16.04.
```
Comment (sbesson) [inline history.rst]: Are we selling this as the main feature of the release or are there other high level improvements worth adding? As such it feels like there is little benefit to upgrading a 5.6.x server on a supported platform for instance.

For Ubuntu 16.04, should mention drop support?

File: history.rst [hunk 2]
```diff
@@ -5,6 +5,24 @@
...
+
+This version has been tested with:
```
Comment (sbesson) [inline history.rst:16]: Unrelated to this PR as i realize it is the wording of all 5.6.x releases. While it makes sense for omero-py and omero-web which are managed separately, these versions of the Java components are shipped with the new bundles.

Maybe we want separate paragraphs in the future.

File: history.rst [hunk 3]
```diff
@@ -5,6 +5,24 @@
...
+
+- omero-blitz 5.5.7
+- omero-gateway-java 5.6.4
+- omero-py 5.7.1
+- omero-web 5.7.0
+- omero-dropbox 5.6.2
```
Comment (sbesson) [inline history.rst:22]: Rfe it might be useful to link to the components changelogs or maybe summarize the major features.

**Review by sbesson — COMMENTED** [review]
A few inline comments. Overall my main uncertainty is how we (under?)promote this release. I will have a look at the release annoucement.

File: history.rst (hunk 3 above)
Comment (jburel) [reply history.rst:22]: Links to the changelogs are in https://github.com/ome/ome-documentation/blob/develop/omero/changelog.rst

File: history.rst (hunk 1 above)
Comment (jburel) [reply history.rst]: Without replicating the content of the changelogs of each "component", this history file is becoming less verbose and less relevant.

Comment (sbesson) [reply history.rst]: Understood. So is the announcement the best reference place to get an overview of all end-users features associated with each release

File: history.rst (hunk 2 above)
Comment (joshmoore) [reply history.rst:16]: I could imagine linking to all the versions (e.g. even omero-model) especially if they were links to the changelogs, though we'd likely want that to be auto-generated.

**Pull request merged**